.github/workflows/greetings.yml

```diff
@@ -7,6 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/first-interaction@v1
+      continue-on-error: true
```

Comment:
**Callout**: Add `continue-on-error` for the time being until issue is [resolved](https://github.com/actions/first-interaction/issues/101).